components/mixins/experience-event/industry-verticals/experienceevent-device-details.schema.json

```diff
@@ -32,20 +32,20 @@
               "title": "Steps",
               "$ref": "https://ns.adobe.com/xdm/datatypes/tool-usage",
               "description": "Extension of toolUsage data type.",
-			  "meta:status": "deprecated"
+              "meta:status": "deprecated"
             },
-            "xdm:deviceTradeInManufacturer": {
+            "xdm:manufacturer": {
               "title": "Manufacturer",
               "type": "string",
               "description": "Manufacturer of the device being traded."
```

Comment:
is this in reference to a traded device anymore? @jbranson000. If not let's remove "being traded."

Reply:
This is still valid and part of the deviceTradeInDetails node, so let's leave as-is for now